lego/apps/events/views.py

```diff
@@ -230,6 +230,9 @@ def create(self, request, *args, **kwargs):
                 'error': f'User {reg.user.username} is not registered for this event'
             })
 
+        if reg.presence != constants.UNKNOWN:
+            raise ValidationError({'error': f'User {reg.user.username} is already present.' })
```

Comment:
Maybe we should send out an error code with these errors as well? I started writing code to handle them specifically in the frontend, and handling the string feels a bit ugly. We probably want to do something like this on a more global level later, but could just do something like `'error_code': 'already_present'` and `'error_code': 'not_registered'` to get started?